components/python/wrapt/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,36 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/lib/python3.7/vendor-packages/wrapt-$(COMPONENT_VERSION)-py3.7.egg-info
+file path=usr/lib/python3.7/vendor-packages/wrapt/__init__.py
+file path=usr/lib/python3.7/vendor-packages/wrapt/_wrappers.cpython-37m.so
+file path=usr/lib/python3.7/vendor-packages/wrapt/decorators.py
+file path=usr/lib/python3.7/vendor-packages/wrapt/importer.py
+file path=usr/lib/python3.7/vendor-packages/wrapt/wrappers.py
+file path=usr/lib/python3.9/vendor-packages/wrapt-$(COMPONENT_VERSION)-py3.9.egg-info
+file path=usr/lib/python3.9/vendor-packages/wrapt/__init__.py
+file path=usr/lib/python3.9/vendor-packages/wrapt/_wrappers.cpython-39.so
+file path=usr/lib/python3.9/vendor-packages/wrapt/decorators.py
+file path=usr/lib/python3.9/vendor-packages/wrapt/importer.py
+file path=usr/lib/python3.9/vendor-packages/wrapt/wrappers.py
```

Comment:
Please also add the generic-manifest (which is more important than the sample-manifest because it's the base for wrapt-PYVER.p5m).
Diff'ing the generic-manifest with your wrapt-PYVER.p5m gives me the following differences:
20c28
< file path=usr/lib/python$(PYVER)/vendor-packages/wrapt/_wrappers.cpython-37m.so
---
> file path=usr/lib/python$(PYVER)/vendor-packages/wrapt/_wrappers.so
24c32,39
< file path=usr/lib/python$(PYVER)/vendor-packages/wrapt/_wrappers.cpython-39.so
---

So seem to miss two generated library files (one for each supported python version). You'll need to add them (with python version guards) like:
$(PYTHON_3.7_ONLY)file path=usr/lib/python$(PYVER)/vendor-packages/wrapt/_wrappers.cpython-37m.so
$(PYTHON_3.9_ONLY)file path=usr/lib/python$(PYVER)/vendor-packages/wrapt/_wrappers.cpython-39.so

How did you add
file path=usr/lib/python$(PYVER)/vendor-packages/wrapt/_wrappers.so

I can't see it in my build.

Reply:
There are default transforms to generate version-specific actions for *.so maybe that's it.

Reply:
@AndWac https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/transforms/python-3-soabi